src/game/main.ts

```diff
@@ -1,31 +1,31 @@
-import { Boot } from './scenes/Boot';
-import { GameOver } from './scenes/GameOver';
-import { Game as MainGame } from './scenes/Game';
-import { MainMenu } from './scenes/MainMenu';
-import { AUTO, Game } from 'phaser';
-import { Preloader } from './scenes/Preloader';
+import { Boot } from "./scenes/Boot";
+import { GameOver } from "./scenes/GameOver";
+import { Game as MainGame } from "./scenes/Game";
+import { MainMenu } from "./scenes/MainMenu";
+import { AUTO, Game } from "phaser";
+import { Preloader } from "./scenes/Preloader";
 
 //  Find out more information about the Game Config at:
 //  https://newdocs.phaser.io/docs/3.70.0/Phaser.Types.Core.GameConfig
 const config: Phaser.Types.Core.GameConfig = {
     type: AUTO,
-    width: 1024,
-    height: 768,
-    parent: 'game-container',
-    backgroundColor: '#028af8',
-    scene: [
-        Boot,
-        Preloader,
-        MainMenu,
-        MainGame,
-        GameOver
-    ]
+    width: 800,
```

Comment:
**suggestion**: Consider using constants for game dimensions to improve maintainability.